R/gather_images.R

```diff
@@ -0,0 +1,84 @@
+#' Gather 1 valid image per species
+#' 
+#' There are multiple images per species, and not all image UIDs are valid.
+#' This function makes searching for a valid image for each species much easier.
+#' Searches for the best name match per \code{species} and 
+#' iteratively tries image UIDs until a viable image is found.
+#' 
+#' Contributed by \href{https://github.com/bschilder}{Brian M. Schilder}.
+#' 
+#' @param species Species list.
+#' @param include_image_data Include the image data itself 
+#' (not just the image UID) in the results.
+#' @param mc.cores Accelerate multiple species queries by parallelising 
+#' across multiple cores.
+#' @param verbose Print messages.
+#' @inheritParams name
+#' @returns data.frame with:
+#' \itemize{
+#' \item{species : }{Species name.}
+#' \item{uid : }{Species UID.}
+#' \item{namebankID : }{\href{http://ubio.org/}{uBio} species ID.}
+#' \item{string : }{Standardised species name.}
+#' \item{picid : }{Image UID.}
+#' }
+#' @source \href{https://github.com/GuangchuangYu/ggimage/blob/master/R/geom_phylopic.R}{
+#' Related function: \code{ggimage::geom_phylopic}}
+#' 
+#' @export 
+#' @examples  
+#' species <- c("Mus_musculus","Pan_troglodytes","Homo_sapiens")
+#' res <- rphylopic::gather_images(species=species)
```

Comment:
please don't namespace calls to the package itself in examples

Reply:
It's not strictly necessary, but also makes it clear where the function comes from in cases where you load another package in the example.

Happy to change it to your preference tho